### ISSUE TITLE
Mutiprotocol support

### ISSUE THREAD

**Issue by ncsky** [FrSky-OS/HORUS-X10-X10S-X12S-bug-report#207]
Dear FrSky,

Could you please implement mutiprotocol module support in FrOS.  As a choice in menu in RF section so that everything is automatically setup for PPM support as well as serial.  User could choose which to use.

Thank you,

Jason

**Comment by ncsky** [comment]
Thank you.

Sent from Yahoo Mail on Android 
 
  On Wed, Aug 1, 2018 at 2:20 AM, Adela<notifications@github.com> wrote:   
It can be achieved after lua is supported when later firmware released.
#211

**Comment by Adelaxie** [comment]
yes and the lua script is under development, please wait for some time.

**Comment by IJK-CZ** [comment]
Any news in Multimodule implementation?
